### ISSUE TITLE
What is the output of pre-trained model, and how it shows the information from the dataset? 

### ISSUE THREAD

**Issue by cfjiang123** [TencentAILabHealthcare/scBERT#11]
What is the output of pre-trained model, and how it shows the information from the dataset? Is the output still kind of expression matrix which shows the information between Genes. I don't really get what the pre-trainning step does.

Besides, I want to know if we don't know the label from the dataset we want to annotate, which means there is no certain/good reference dataset. Which dataset could be used to fine-tune?
